specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2022-10-31-preview/iotcentral.json

```diff
@@ -0,0 +1,9326 @@
+      "x-ms-discriminator-value": "command",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TileConfiguration"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "group": {
+          "description": "The ID of the device group to display",
+          "type": "string"
+        },
+        "command": {
+          "description": "The command to reference in the tile",
+          "type": "string"
+        },
+        "device": {
+          "description": "The device to reference in the tile"
+        }
+      },
+      "required": [
+        "group",
+        "command",
+        "device"
+      ]
+    },
+    "CountQueryRangeConfiguration": {
+      "description": "Configuration specifying the number of data points to query for a tile.",
+      "x-ms-discriminator-value": "count",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/QueryRangeConfiguration"
+        }
+      ],
+      "properties": {
+        "count": {
+          "description": "The maximum number of data points to query for.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 100,
+          "maximum": 100,
+          "default": 100
+        }
+      },
+      "required": [
+        "count"
+      ]
+    },
+    "Dashboard": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Unique ID of the dashboard.",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "Display name of the dashboard.",
+          "type": "string",
+          "minLength": 1
+        },
+        "tiles": {
+          "description": "The tiles displayed by the dashboard.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Tile"
+          },
+          "maxItems": 100
+        },
+        "personal": {
+          "description": "Whether the dashboard is personal and can only be viewed by the current user.",
+          "type": "boolean",
+          "default": false,
+          "readOnly": true
+        },
+        "favorite": {
+          "description": "Whether the dashboard is favorited or not",
+          "type": "boolean",
+          "default": false
+        },
+        "etag": {
+          "description": "Etag to prevent conflict when updating the dashboard.",
+          "type": "string"
+        },
+        "organizations": {
+          "description": "The organization the dashboard belongs to. If not present, the dashboard is root-level or personal. only one organization is supported today, multiple organizations will be supported soon.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "displayName"
+      ]
+    },
+    "DashboardCollection": {
+      "description": "The paged results of dashboards.",
```

Comment:
Nit: Above, I noticed you changed a description like this containing "page results of" to just "collection of". Either seems fine, but I'd be consistent across the API.

Reply:
We made [Entity]Collection description as "the paged results" and the value property as "collection of"